validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/graffiti/GraffitiLoader.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.client.loader.graffiti;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.apache.tuweni.bytes.Bytes32;
+
+public class GraffitiLoader {
+  public static Bytes32 loadFromFile(final Path graffitiFile) throws GraffitiLoaderException {
+    checkNotNull(graffitiFile, "GraffitiFile path cannot be null");
+    try {
+      return Bytes32Converter.toBytes32(Files.readAllBytes(graffitiFile));
```

Comment:
Rather than using `readAllBytes`, it's probably safer to use something like `FileInputStream` to read at most 32 bytes, and then detect if there's extra unread data and, if so, throw an error.

Reply:
Great idea. What if we just read in the first 32 and ignored the rest?

Thinking out loud... I think that would error in the case of users adding 32 chars and a new line char?

Reply:
That's a good point.  I think to start, I'd err on the side of being strict (fail to read from the file if the format is unexpected).  This could help prevent errors where the node operator has pointed to the wrong file, for example.  We want to be pretty careful about what we post to the chain - no way to remove it later :D

Reply:
We could always add some validation at startup - try to read from the provided file and if it fails, refuse to start.  This would give the node operator a heads up that the format is wrong.

Reply:
I've put a stricter check in. Of course we can't rely on failing at start up because the file can change subsequently. Let me know what you think.